string[] ids
furniture/Pose_2D_With_Z[] poses